.github/workflows/pullrequest_check.yml

```diff
@@ -59,15 +61,14 @@ jobs:
         run: yarn run check
 
   integration_and_unit_tests:
-    if: github.event.pull_request.draft == false
```

Comment:
I don't think this line should be removed here. 
With this line removed the normal tests will run none cli related draft PRs.

Reply:
That was so silly. I removed it by mistake. I have put that back in.